### PR TITLE
Fix reactivity in the editor components and various Vue 3 related fixes

### PR DESCRIPTION
**Pull request by JohanAhlen**

### CODE REVIEW

File: desktop/core/src/desktop/js/apps/editor/components/ExecutableActionsKoBridge.vue [hunk 1]
```diff
@@ -49,39 +48,19 @@
...
 
-    data(): {
-      initialized: boolean;
-      executable: SqlExecutable | null;
-    } {
       return {
-        initialized: false,
-        executable: null
+        executable,
+        subTracker
```
Comment (sreenaths) [inline desktop/core/src/desktop/js/apps/editor/components/ExecutableActionsKoBridge.vue:61]: Nit: This need not be returned right?

File: desktop/core/src/desktop/js/apps/editor/components/aceEditor/AceEditorKoBridge.vue [hunk 2]
```diff
@@ -76,83 +75,45 @@
...
+      subTracker.trackObservable(cursorPositionObservable, cursorPosition);
+      subTracker.trackObservable(valueObservable, value);
+
       return {
+        cursorPosition,
+        editorId,
         sqlParserProvider: sqlParserRepository,
         sqlReferenceProvider: sqlReferenceRepository,
-        subTracker: new SubscriptionTracker()
+        subTracker,
```
Comment (sreenaths) [inline desktop/core/src/desktop/js/apps/editor/components/aceEditor/AceEditorKoBridge.vue:95]: Nit: This need not be returned right?

**Review by sreenaths — APPROVED** [review]
Nice. trackObservable looks cool.

File: desktop/core/src/desktop/js/apps/editor/components/aceEditor/AceEditorKoBridge.vue (hunk 2 above)
Comment (JohanAhlen) [reply desktop/core/src/desktop/js/apps/editor/components/aceEditor/AceEditorKoBridge.vue:95]: Indeed subTracker can be removed here.

File: desktop/core/src/desktop/js/apps/editor/components/ExecutableActionsKoBridge.vue (hunk 1 above)
Comment (JohanAhlen) [reply desktop/core/src/desktop/js/apps/editor/components/ExecutableActionsKoBridge.vue:61]: Yes. Will drop.

**Pull request merged**